arch/arm64/boot/dts/qcom/sm8475-samsung-q4q.dts

```diff
@@ -0,0 +1,738 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2024, The Linux Foundation. All rights reserved.
+ */
+
+/dts-v1/;
+
+#include <dt-bindings/arm/qcom,ids.h>
+#include <dt-bindings/regulator/qcom,rpmh-regulator.h>
+#include <dt-bindings/leds/common.h>
+
+#include "sm8475.dtsi"
+#include "pm8350.dtsi"
+#include "pm8350b.dtsi"
+#include "pm8350c.dtsi"
+#include "pm8450.dtsi"
+#include "pmk8350.dtsi"
+#include "pmr735a.dtsi"
+
+/ {
+	model = "Galaxy Z Fold4";
+	chassis-type = "phablet";
+
+	qcom,msm-id = <530 0x10000 0>;
```

Comment:
qcom,msm-id = <QCOM_ID_SM8475  0x10000 0>;

Reply:
I assume that qcom,msm-id = <QCOM_ID_SM8475 0x10000>; will work too

Reply:
yep, thanks